Automattic/jetpack
# Set .gitattributes text=auto for all files

#### Changes proposed in this Pull Request:

This will hopefully prevent people from accidentally converting files from LF to CRLF line endings. On Windows, all files checked out should be converted to CRLF line endings, then converted back on commit.

Also normalize a few files that currently have CRLF endings and shouldn't anymore.

#### Jetpack product discussion


https://github.com/Automattic/jetpack/pull/18061#issuecomment-743342000

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Check things out in different environments, then make changes and push some commits and see that things aren't getting screwed up.

#### Proposed changelog entry for your changes:


* None needed.
